### ISSUE TITLE
Adjustments to Raumfrei Extra

### ISSUE THREAD

**Issue by Schedulaar** [Integreat/app-cross_platform#421]
After meeting with Neuburg, the following changes still need to be made:
* Display base rent black (for accessibility)
* Instead of display "0€"-costs, display "Keine"
* Offers should be sorted by creationDate (latest at the top)
* Display Title instead of landlord-name in overview
* Display Title and Location  (new fields!) in DetailView
* Allow "Wohnzimmer" (living-room) in the list of available rooms
